src/main/java/com/minelittlepony/ducks/IPlayerInfo.java

```diff
@@ -0,0 +1,17 @@
+package com.minelittlepony.ducks;
+
+import net.minecraft.client.network.NetworkPlayerInfo;
+
+public interface IPlayerInfo {
+    /**
+     * Returns true if the vanilla skin (the one returned by NetworkPlayerInfo.getSkinLocation) uses the ALEX model type.
+     */
+    boolean usesSlimArms();
+
+    /**
+     * Quick cast back to the original type.
+     */
+    default NetworkPlayerInfo unwrap() {
+        return (NetworkPlayerInfo)this;
```

Comment:
This is only used in mixins, so you can replace it with a double cast. `(NetworkPlayerInfo) (Object) this`

Reply:
Perhaps, but I never liked the look of a double cast.